pages/blog/hackathon-faq.md

```diff
@@ -0,0 +1,98 @@
+excerpt: Answers to all the questions you ever wanted to ask about the AsyncAPI Hackathon 2021
+featured: true
+---
+
+Last year it was the first time we hosted the AsyncAPI Conference. This year it is time to host our first AsyncAPI Hackathon. Many questions are coming from the community, and this article tries to address them.
+
+### When is the Hackathon?
+
+1st - 31st of October 2021.
+
+You have an entire month to complete your idea. No usual over-weekend hard work but relaxed month instead.
+
+### What is the main theme for the Hackathon?
+
+The goal of the Hackathon is to provide MVP solutions that can help AsyncAPI Community in ways like:
+
+- Ease education and getting started with the project.
+- Bring new AsyncAPI-related solutions that can solve the existing challenges the community is facing with event-driven architectures.
+- Extend existing tools, reuse them in some scenarios, add the extra implementation on top.
+
+### Do I need to register for the event?
+
+No.
+
+Anybody can join the Hackathon. Whenever you want. No pressure in case you can't deliver what you planned.
+
+### Is it team competition or for individuals?
+
+Both.
+
+You can form a team if you want. Just keep in mind that once you win, the prize also goes to the entire team.
+
+### I want to participate, but I don't have a good idea
+
+No worries, look at ideas that [others shared](https://github.com/asyncapi/community/discussions/categories/asyncapi-hack-2021-ideas-brainstorming), you can pick one of them or just get some inspiration.
+
+### Can there be multiple submissions with the same idea?
+
+Yes.
+
+We do not set any limits here. The same idea can be done by multiple teams. Just keep in mind that if there are two excellent submissions for the same idea, I doubt judges will vote for both to let other ideas shine, although this is not an official rule.
+
+### What is the scope of the submission? What does it mean I finished my solution on time?
+
+- Your solution must be fully open-sourced and stored in a repository under your GitHub profile. Please do not use Bitbucket or GitLab as it will increase complexity for our community members that will review submissions.
+- It cannot integrate with a commercial solution that has no free tier.
+- You must submit it before the deadline. Evaluation of the solution will not take into account any commits pushed to the repository after the deadline.
+
+### How do I submit my task?
+
+Create a new discussion item [here](https://github.com/asyncapi/community/discussions/categories/asyncapi-hack-2021-submissions). Make sure there is:
+- Overview of what the solution is about.
+- Screenshots/Video/Diagrams/Code snippets (`anyOf` :wink:) that helps judges to quickly understand the solution and encourage them for further review.
+- Link to the repository where the full solution can be found.
+
+This is not it. If you win, you need to present your idea at [AsyncAPI Conference in November](http://conference.asyncapi.com/)
```

Comment:
```suggestion
If you win, you get to present your idea at our upcoming 2021 [AsyncAPI Conference in November](http://conference.asyncapi.com/)
```

I think it accidentally sounds a little short/rude to start with "this is not it." That's why I removed it. Also overall was trying to make things sound friendlier.